bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/model/SomfyTahomaAction.java

```diff
@@ -26,7 +27,7 @@
 public class SomfyTahomaAction {
 
     private String deviceURL = "";
-    private ArrayList<SomfyTahomaCommand> commands = new ArrayList<SomfyTahomaCommand>();
+    private List<SomfyTahomaCommand> commands = new ArrayList<SomfyTahomaCommand>();
```

Comment:
Sorry, didn't see that. Use diamond operator where possible

```
     private List<SomfyTahomaCommand> commands = new ArrayList<>();
```

Reply:
I have got rid of the obsolete SomfyTahomaCommand. thanks